tests/conda_env/support/requirements.txt

```
@@ -1 +1 @@
-flask==1.1.1
+flask==2.3.3
```

Let's use a package that we have control over and that is unlikely to receive a CVE, or at least then it's on us to fix this test.

```suggestion
conda-package-handling==2.2.0
```

This'll also need an update of line https://github.com/jmcarpenter2/conda/blob/32e4661243cf863f17b71bad763dfc2e6630bf02/tests/conda_env/specs/test_requirements.py#L27

---

Do those tests even check that the package is real or exists in the index?

Looks like a good choice as long as the environment under test wouldn't automatically get conda-package-handling as a conda dependency.

---

Only `test_environment` actually tests the parsing, the others only if the file is there and valid, which could be any matchspec compatible string IIUC